packages/delisp-core/src/compiler/inline-primitives.ts

```diff
@@ -175,3 +200,17 @@ defineInlinePrimitive("length", "(-> [a] number)", ([vec]) => {
     property: { type: "Identifier", name: "length" }
   };
 });
+
+// matches `.foo` and inlines `(-> {foo a | b} a)`
+defineMagicPrimitive(
+  name => name[0] === ".",
+  name => {
+    const label = name.slice(1);
+    return createInlinePrimitive(`(-> {${label} a} a)`, ([vec]) => ({
```

Comment:
Can we change the first argument so it's not a string but a `Type`?   Then we don't need to use the fragile string interpolation here but just `tRecord`.

Reply:
The record label is a string, though. What Type would be passed in here? The Symbol?

Reply:
Aah wait completely misunderstood which argument you meant. Sure we can do that, though if it would work differently from other primitives then. Should we support `Type | string` maybe?

Reply:
Sure, or we can have a lower-level  `createPrimitive` (type), and a `createEasyPrimitive` (string) on top of it.